components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -233,6 +206,33 @@
 <config_cvmix_kpp_use_enhanced_diff>.true.</config_cvmix_kpp_use_enhanced_diff>
 <config_cvmix_kpp_nonlocal_with_implicit_mix>.false.</config_cvmix_kpp_nonlocal_with_implicit_mix>
 
+<!-- vmix_const -->
+<config_use_const_visc>.false.</config_use_const_visc>
+<config_use_const_diff>.false.</config_use_const_diff>
+<config_vert_visc>1.0e-4</config_vert_visc>
+<config_vert_diff>1.0e-5</config_vert_diff>
+
+<!-- vmix_rich -->
+<config_use_rich_visc>.false.</config_use_rich_visc>
+<config_use_rich_diff>.false.</config_use_rich_diff>
+<config_bkrd_vert_visc>1.0e-4</config_bkrd_vert_visc>
+<config_bkrd_vert_diff>1.0e-5</config_bkrd_vert_diff>
+<config_rich_mix>0.005</config_rich_mix>
+
+<!-- vmix -->
+<config_convective_visc>1.0</config_convective_visc>
+<config_convective_diff>1.0</config_convective_diff>
+
+<!-- vmix_tanh -->
+<config_use_tanh_visc>.false.</config_use_tanh_visc>
+<config_use_tanh_diff>.false.</config_use_tanh_diff>
+<config_max_visc_tanh>2.5e-1</config_max_visc_tanh>
+<config_min_visc_tanh>1.0e-4</config_min_visc_tanh>
+<config_max_diff_tanh>2.5e-2</config_max_diff_tanh>
+<config_min_diff_tanh>1.0e-5</config_min_diff_tanh>
+<config_zMid_tanh>-100</config_zMid_tanh>
+<config_zWidth_tanh>100</config_zWidth_tanh>
+
```

Comment:
should this be changed?

Reply:
none of these settings are changed -- the same things are removed above. But the Registry file must have been shuffled and the auto-generation scripts just follow that for the ordering of the namelists